### PR TITLE
Fabo/0.33.0

### LINKED ISSUE #2103: Undelegations changed format
The new format is:

```
[
  {
    "delegator_addr": "cosmos14l8etu9dx5c3v5cv6pgwa824m4v3c3w65f86q0",
    "validator_addr": "cosmosvaloper1fj9v03uf3wx805m6y6q3flfwgyt9xw82sswaws",
    "entries": [
      {
        "creation_height": "964",
        "completion_time": "2019-03-01T16:16:21.41396Z",
        "initial_balance": "10000000",
        "balance": "10000000"
      }
    ]
  }
]
```

### PR DESCRIPTION
**Pull request by faboweb**
Closes #2103 

Updates to SDK 0.33.0
Improves on local testnet generation and usage. Always sends tokens to a fixed address: "cosmos1ek9cd8ewgxg9w5xllq9um0uf4aaxaruvcw4v9e"

Needs the backend to be updated

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `PENDING.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: app/src/renderer/components/transactions/TmLiAnyTransaction.vue [hunk 1]
```diff
@@ -0,0 +1,100 @@
+<template>
```
Comment (fedekunze) [inline app/src/renderer/components/transactions/TmLiAnyTransaction.vue]: We renamed this file to LiAnyTransaction. Why is this being changed ?

**Comment by faboweb** [comment]
It's not. Merge conflict. I removed the file again.

Federico Kunze <notifications@github.com> schrieb am Sa., 9. März 2019,
19:31:

File: app/src/renderer/scripts/time.js [hunk 2]
```diff
@@ -2,13 +2,13 @@
 
 const getUnbondingTime = ({ height, tx }, unbondingDelegations) => {
   const { type, value } = tx.value && tx.value.msg && tx.value.msg[0]
-  if (type === `cosmos-sdk/Undelegate`) {
-    const unbondingDelegation = unbondingDelegations[value.validator_addr]
+  if (type === `cosmos-sdk/MsgUndelegate`) {
+    const validatorUnbondingDelegation = unbondingDelegations[value.validator_address]
```
Comment (jbibla) [inline app/src/renderer/scripts/time.js]: is `validatorUnbondingDelegation` being used?

the naming is a bit confusing too. 

`validatorUnbondingDelegation = unbondingDelegations[value.validator_address]`
`unbondingDelegation = validatorUnbondingDelegation.find()`

File: app/src/renderer/vuex/modules/delegation.js [hunk 3]
```diff
@@ -159,11 +156,11 @@ export default ({ node }) => {
 
       await dispatch(`sendTx`, {
         type: `postDelegation`,
-        to: wallet.address, // TODO strange syntax
+        to: session.address, // TODO strange syntax
```
Comment (jbibla) [inline app/src/renderer/vuex/modules/delegation.js]: still strange syntax?

File: package.json [hunk 4]
```diff
@@ -36,7 +36,7 @@
     "prepush": "bash ./tasks/changelog-changed-check.sh && yarn lint",
     "postcheckout": "yarn",
     "watch": "tasks/watch.sh",
-    "nodes": "node ./tasks/testnet.js",
+    "nodes": "node ./tasks/local-testnet/start.js",
```
Comment (jbibla) [inline package.json]: 👍

File: app/src/renderer/vuex/modules/delegation.js (hunk 3 above)
Comment (faboweb) [reply app/src/renderer/vuex/modules/delegation.js]: yep, just changed the source of the address

Comment (fedekunze) [reply app/src/renderer/vuex/modules/delegation.js]: what's the strange syntax? can we create an issue ?

**Comment by fedekunze** [comment]
hmm I still get the error from undelegations: ` Error fetching delegations: Cannot read property 'amount' of undefined`

File: app/src/renderer/vuex/modules/delegation.js (hunk 3 above)
Comment (faboweb) [reply app/src/renderer/vuex/modules/delegation.js]: there is already an issue around refactoring the send module #1828

**Comment by faboweb** [comment]
@fedekunze could you re-eval please?

**Comment by fedekunze** [comment]
> none of the actions work

This is still failing: (re)delegation, unbonding, proposals, send, etc

**Comment by faboweb** [comment]
> This is still failing: (re)delegation, unbonding, proposals, send, etc

will fix this

**Comment by faboweb** [comment]
Need to check version again. The commit stated 0.32.0. Maybe need to upgrade again.

**Comment by fedekunze** [comment]
Karoly was this manually tested ?

**Comment by fedekunze** [comment]
@sabau @faboweb this wasn't properly tested and it broke the transactions page

**Pull request merged**